modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/ODBProxy.scala

```diff
@@ -31,7 +31,7 @@ class ODBProxy(val loc: Peer, cmds: ODBProxy.OdbCommands) {
 
   def host(): Peer = loc
   def read(oid: SPObservationID): SeqexecFailure \/ ConfigSequence =
-    SeqExecService.client(loc).sequence(oid).leftMap(SeqexecFailure.ODBSeqError)
+    SeqExecService.client(loc).sequence(oid).map(_.config).leftMap(SeqexecFailure.ODBSeqError)
```

Comment:
Is this change part of it?

Reply:
Yes, I changed the return type of `sequence` from `ConfigSequence` to `SeqexecSequence` which contains more data including the sequence name and the already executed file ids. Those will be used in a future PR but this config sequence needs to be extracted here in any case